src/utils/transforms.lisp

```diff
@@ -500,6 +503,9 @@
   "Convert a DB3 logical value to a PostgreSQL boolean."
   (if (member value '("?" " ") :test #'string=) nil value))
 
+(defun progress-logical-to-boolean (value)
+    (if (string= value "yes") "1" (if (string= value "no") "0" nil))
+)
```

Comment:
Please close all the parens on the same line. In Lisp closing parens have no semantics attached to them.